python/ribasim/ribasim/model.py

```diff
@@ -170,9 +181,18 @@ def _save(self, directory: DirectoryPath, input_dir: DirectoryPath):
         for sub in self._nodes():
             sub._save(directory, input_dir)
 
+    def set_crs(self, crs: str) -> None:
```

Comment:
I would expect this method (only?) set `self.crs = crs`, but it doesn't. If I do model.crs = "something else", it would overwrite anything set in this method on _save.